### ISSUE TITLE
Noma doesn't stop all services

### ISSUE THREAD

**Issue by nolim1t** [lncm/noma#55]
When doing a ```noma stop``` only LND stops but not invoicer.

So Ideally it should go into the ```docker-compose``` directory and issue a ```docker-compose down``` after LND is stopped.

This should also be a fallback incase LND does not stop for some reason

**Comment by AnotherDroog** [comment]
Good idea. 

Have you noticed some differences between `docker stop` and `docker-compose down`?

There’s also `docker kill` for stuck processes

**Comment by nolim1t** [comment]
yep it should just force quit all the processes eventually if stuff gets "stuck"

**Comment by nolim1t** [comment]
But the main issue is invoicer currently doesn't stop when you do ```noma stop```
